src/components/verification-grid/verification-grid.ts

```diff
@@ -310,25 +310,9 @@ export class VerificationGridComponent extends AbstractComponent(LitElement) {
       return;
     }
 
-    if (event.ctrlKey && event.key === "a") {
-      // we prevent default on the ctrl + A event so that chrome doesn't
-      // select all the text on the page
-      event.preventDefault();
-
-      this.subSelectAll();
-      // we return early after every keyboard shortcut because we know that
-      // we can't trigger multiple keyboard shortcuts at the same time
-      // e.g. You can't press ctrl + A and escape to select and deselect
-      // by returning early we save computing erroneous keyboard shortcuts
-      return;
-    }
-
-    if (event.ctrlKey && event.key === "d") {
-      event.preventDefault();
-      this.removeSubSelection();
-    }
-
     switch (event.key) {
+      // the escape button is a native browser shortcut to toggle fullscreen
+      // mode. We therefore do not want to bind to the escape key
```

Comment:
but it's still bound?

Reply:
The comment is incorrect

<https://stackoverflow.com/a/37759224>

I've removed the comment and left the code unchanged